portal-web/test/functional/com/liferay/portalweb/macros/headless/headlessadmintaxonomy/TaxonomyVocabularyAPI.macro

```diff
@@ -112,6 +184,28 @@ definition {
 		return "${response}";
 	}
 
+	macro getTaxonomyVocabularyWithDifferentParameters {
+		var curl = TaxonomyVocabularyAPI._curlTaxonomyVocabulary(
+			assetLibraryId = "${assetLibraryId}",
+			parameter = "${parameter}",
+			parameterValue = "${parameterValue}",
+			siteId = "${siteId}");
+
+		var response = JSONCurlUtil.get("${curl}");
+
+		return "${response}";
+	}
+
+	macro setUpGlobalSiteId {
+		var siteId = JSONGroupAPI._getGroupIdByNameNoSelenium(
+			groupName = "Guest",
```

Comment:
Just a little curious that the name is "setUpGlobalsiteId" but we're using "Guest" site not using "Global" site here.